data/datavalid.yml

```diff
@@ -402,15 +398,9 @@ schemas:
       - name: appealed
       - name: agency
       - name: motions
-      - name: appeal_disposition_year
-      - name: appeal_disposition_month
-      - name: appeal_disposition_day 
-      - name: appeal_hearing_year
-      - name: appeal_hearing_month
-      - name: appeal_hearing_day
-      - name: appeal_hearing_2_year 
-      - name: appeal_hearing_2_month
-      - name: appeal_hearing_2_day
+      - name: appeal_disposition_date
+      - name: appeal_hearing_date
+      - name: appeal_hearing_2_date
```

Comment:
Since we already have the events, we don't need these date columns anymore do we?

Reply:
Good point. @rajivsinclair I made these changes based on our discussion yesterday re the exclusion of dates in the appeals table. Thoughts?

Reply:
Sorry for any confusion. Are there more than one type of date for each appeal event? If so, only the primary date field should appear in the event table. The other date fields could be stored here in this table and attributes of the event.

Reply:
Let's discuss this further.

Reply:
Sorry for the confusion/back-and-forth on this, but upon further discussion and thought, we decided that each date should be a separate event (with a different `event_kind`) and these events should be linked together by a common `appeal_uid`. This way we will be consistent with the existing method for handling of multiple dates that are already associated with a single record as with `allegation` data.